### PR TITLE
Update Menu UI

### PR DESCRIPTION
**Pull request by donnaqi**
- Add a new sound setting UI for music and SFX
- Add a graphic of the keyboard controls UI
- Both can be accessed from the pause menu
- Very minor change: press "esc" can only open pause menu, if need to resume to the game, need to press "RESUME" button and press "esc" no longer work.
- Within Sound and Control settings, the player can press "Back" to return to the pause menu, and "EXIT" to leave the settings directly

#109 

Screenshot 2024-03-17 at 4 57 24 PM
Screenshot 2024-03-17 at 4 57 29 PM
Screenshot 2024-03-17 at 4 57 35 PM

### CODE REVIEW

File: Assets/Scripts/UI/PauseMenu.cs [hunk 1]
```diff
@@ -22,22 +28,27 @@ void Start()
...
+
+        //     if (GameIsPaused)
+        //     {
+        //         ResumeGame();
+        //     }
+        //     else
+        //     {
+        //         PauseGame();
+        //     }
+        // }
```
Comment (wang-henry) [inline Assets/Scripts/UI/PauseMenu.cs:42]:
```suggestion
```

File: Assets/Scripts/UI/PauseMenu.cs [hunk 2]
```diff
@@ -22,22 +28,27 @@ void Start()
...
-                PauseGame();
-            }
+            PauseGame();
         }
     }
 
     public void ResumeGame()
     {
-        GameIsPaused = false;
+        // GameIsPaused = false;
```
Comment (wang-henry) [inline Assets/Scripts/UI/PauseMenu.cs:51]:
```suggestion
```

File: Assets/Scripts/UI/PauseMenu.cs [hunk 3]
```diff
@@ -53,7 +64,7 @@ void PauseGame()
     {
         HUDDisableEvent.TriggerEvent();
         Time.timeScale = 0f;
-        GameIsPaused = true;
+        // GameIsPaused = true;
```
Comment (wang-henry) [inline Assets/Scripts/UI/PauseMenu.cs:67]:
```suggestion
```

File: Assets/Scripts/UI/PauseMenu.cs [hunk 4]
```diff
@@ -72,4 +83,55 @@ public void QuitGame()
...
+
+    public void ExitfromSoundSetting()
+    {
+        soundMenuUI.SetActive(false);
+        if (!loginPanelUI.activeInHierarchy && !controlPanelUI.activeInHierarchy)
+        {
+            HUDEnableEvent.TriggerEvent();
+            Time.timeScale = 1f;
+            Cursor.lockState = CursorLockMode.Locked;
+            Cursor.visible = false;
```
Comment (wang-henry) [inline Assets/Scripts/UI/PauseMenu.cs:115]: Can probably be made into a new function so that it can be reused in `ExitfromControlMenu`.

**Review by wang-henry — APPROVED** [review]

File: Assets/Scripts/UI/PauseMenu.cs (hunk 4 above)
Comment (donnaqi) [reply Assets/Scripts/UI/PauseMenu.cs]: [3872a39](https://github.com/ectodrop/loop-game/pull/112/commits/3872a3979d75229190167d87512711d71e7aab0b)

**Comment by wang-henry** [comment]
Looks good to me!

**Comment by ectodrop** [comment]
👍

**Pull request merged**